1Mininrotatedsortedarraybf.java

```diff
@@ -0,0 +1,11 @@
+
+class Solution {
+    public int findMin(int[] nums) {
+        int min = nums[0];
+        for(int i=1;i<nums.length;i++){
+            if(nums[i]<min)
+                min = nums[i];
+        }
+        return min;
+    }
+}
```

Comment:
Try solving using 2 pointer approach for an optimized solution. Refer to sample answer.